nginx-proxy/nginx.conf

```diff
@@ -0,0 +1,35 @@
+load_module modules/ngx_http_js_module.so;
```

Comment:
I'm guessing these can't go inside the conf.d includes and therefore inside the nginx.tmpl, right?

If I take a look at https://github.com/jwilder/nginx-proxy/blob/c33dedf10be03d31b73ac9b3c3910613ea886e03/Dockerfile#L14 - nginx-proyx "injects" directives into the default nginx.conf, so I wonder if we should do something similar here, rather than overwriting the entire file? (although in many ways, this is much easier to read and understand ... Just need to catch-up if the parent nginx-proxy changes I suppose)

Reply:
Also, I'm actually not sure I understand why we need to calculate the hash at all... Can't we just check if the file exists in our cache? i.e. just change the folder from `v2` to `defatult` and things would just work??

Reply:
`I'm guessing these can't go inside the conf.d includes and therefore inside the nginx.tmpl, right?`
Yep, load_module modules/ngx_stream_js_module.so; so I've had to copy the whole nginx.conf.